endpoints/users_id_token.py

```diff
@@ -219,7 +221,9 @@ def _maybe_set_current_user_vars(method, api_info=None, request=None):
       _set_bearer_user_vars(allowed_client_ids, scopes)
 
 
-def _get_token(request):
+def _get_token(
+    request=None, allowed_auth_schemes=('OAuth', 'Bearer'),
```

Comment:
I think it's worth type-checking these parameters to ensure that they're iterable (since they're both used in a "for _ in _" loop).